### PR TITLE
Sean's changes to bgp

### PR DESCRIPTION
**Pull request by isharacomix**
Addresses #2, #3, #4, #5, #6

### CODE REVIEW

File: bgp-numbered-ipv6/leaf02/Quagga.conf [hunk 1]
```diff
@@ -1,10 +1,12 @@
 !
 router bgp 65012
+  bgp default show-hostname
   no bgp default ipv4-unicast
   bgp router-id 10.0.0.12
   maximum-paths 64
   bgp bestpath as-path multipath-relax
   bgp bestpath compare-routerid
+  bgp bestpath import-check exact
```
Comment (isharacomix) [inline bgp-numbered-ipv6/leaf02/Quagga.conf]: This is pretty slick

**Comment by isharacomix** [comment]
Verified that all 8 demos still work on reference topology.

File: bgp-numbered-ipv6/leaf04/Quagga.conf [hunk 2]
```diff
@@ -1,10 +1,12 @@
 !
 router bgp 65014
+  bgp default show-hostname
   no bgp default ipv4-unicast
   bgp router-id 10.0.0.14
   maximum-paths 64
   bgp bestpath as-path multipath-relax
   bgp bestpath compare-routerid
+  bgp bestpath import-check exact
```
Comment (jaheller) [inline bgp-numbered-ipv6/leaf04/Quagga.conf]: This should be: bgp network import-check exact

File: bgp-numbered-ipv6/leaf01/Quagga.conf [hunk 3]
```diff
@@ -1,10 +1,12 @@
 !
 router bgp 65011
+  bgp default show-hostname
   no bgp default ipv4-unicast
   bgp router-id 10.0.0.11
   maximum-paths 64
   bgp bestpath as-path multipath-relax
   bgp bestpath compare-routerid
```
Comment (ericpulvino) [inline bgp-numbered-ipv6/leaf01/Quagga.conf]: I know this line is everywhere but we should remove... somehow this got into the default config but it should not be there. Provides no benefit in ebgp ECMP networks.

File: bgp-numbered-ipv6/server01/interfaces [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
 auto eth0
 iface eth0 inet dhcp
     post-up ip route add default via 192.168.0.1
 
 auto eth1
 iface eth1 inet6 static
-    address fd16::1:101
-    netmask 112
+    address fd16::1:101/112
     pre-up sysctl net.ipv6.conf.eth1.accept_dad=0
```
Comment (ericpulvino) [inline bgp-numbered-ipv6/server01/interfaces]: Stylistic comment but adding it none the less
I would typically write "pre-up sysctl net.ipv6.conf.eth1.accept_dad=0" as "pre-up sysctl net.ipv6.conf.$IFACE.accept_dad=0"
Config is more portable.

File: bgp-numbered-ipv6/leaf01/Quagga.conf (hunk 3 above)
Comment (ddutt) [inline bgp-numbered-ipv6/leaf01/Quagga.conf]: I presume you're talking about the compare-routerid piece, and if so, I
agree,

Dinesh

File: bgp-numbered-ipv6/server01/interfaces (hunk 4 above)
Comment (isharacomix) [inline bgp-numbered-ipv6/server01/interfaces]: Can you extract that into its own issue? I'll address it next time I move through the demo.

**Pull request merged**